### DISCUSSION TITLE
Symbols Nerd Font Mono smaller sizing

### DISCUSSION THREAD

**Discussion by trevarj** [ryanoasis/nerd-fonts#1496] [Q&A] [answered]
Hi, I noticed from some old GH issues that a while ago there were multiple sizes for the "Symbols Only" ttf (at least there were some old Arch AUR packages with that). Is it possible to build the symbols only with a slightly smaller size? Can someone tell a small brain loser like me how it can be done? Then I will push the buttons on my computer and watch the CPU usage get to 100% 🚀 

My reasoning is because I am using Iosevka (customized terminal variant) and, due to its condensed width, the fallback glyphs appear large and too close to the next cell in my terminal (Kitty). I'm pretty sure rendering the symbols slightly smaller will make this annoying issue go away for me and free me of tinkering for a few weeks. Thanks.

Example:
![image](https://github.com/ryanoasis/nerd-fonts/assets/5448324/e93318dd-4776-4cec-9d4a-f935c40fc977)

**Comment by trevarj** [comment, marked as answer]
I actually just fired up fontforge and did a quick transform on the patched font to make all glyphs 75% in size and it's perfect.

1. Load patched-fonts/NerdFontSymbolsOnly/SymbolsNerdFontMono-Regular.ttf
2. Ctrl-A to select all glyphs
3. Ctrl-\ to open transformations (Element -> Transformations -> Transform)
4. Select "Scale Uniformly" 
5. Enter desired size (75% for me)
6. Ok
7. File -> Generate Fonts...

**Comment by Finii** [comment]
> were multiple sizes for the "Symbols Only" ttf (at least there were some old Arch AUR packages with that)

We had two different units=per=EM (1000 and 2048) but with the same icon size. Kind of ridiculous because the "EM size" is just a font internal definition of unit.

![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/3b071c53-8229-4577-bd58-ee00d123d197)
_Excerpt of Haralambous' Fonts&Encodings_

**Reply by trevarj** [reply]
Thanks for the info!
